wordpress-mobile/AztecEditor-iOS
# Reorganizes the project a bit.  Adds AttributeType.

### Description:

This PR reorganizes the project a bit and adds `AttributeType` to avoid using literal constants for the most common attribute names.  `AttributeType` is extensible by 3rd party Apps.

### Testing:

1. Run the unit tests
2. Launch the editor
3. Go to HTML mode and add a tag with any of these: `href`, `rel`, `style`, `target`, `src`, `class`.
4. Switch to visual mode.
5. Switch back to HTML mode and make sure the attribute is sitll there.
